classes/Project/Project.mjs

```diff
@@ -1,33 +1,36 @@
 import dbDriver from "../../database/driver.mjs"
-import { validateProjectPayload } from "../../utilities/validatePayload.mjs"
+import {validateProjectPayload} from "../../utilities/validatePayload.mjs"
 import {User} from "../User/User.mjs"
 
 let err_out = Object.assign(new Error(), {
   status: 500,
-  message: "Unknown Server error",
- })
+  message: "Unknown Server error"
+})
 
 const database = new dbDriver("mongo")
 
 export default class Project {
+  #creator
   constructor(userId) {
-    this.creator = userId
+    this.id = userId
```

Comment:
I'm not quite sure I get this...
A `new Project()` would always need a `userId` to invoke. What if I am a Project contributor and want to instantiate the class to do something like `getByID()` or some team modification? I know JS Classes are not exactly Java ones so this can sort of work, but I'm nervous.